src/libs/DockerApi.ts

```diff
@@ -101,6 +103,10 @@ export default class DockerApi {
     return request(requestOptions)
   }
 
+  getVersion() {
```

Comment:
Make the property `version`public and readonly, thus this getter is not required anymore.
`private version: string` => `public readonly version: string`